package com.personal.ai.chatbot.exceptions;


public class UserNotFoundException extends Exception {

     public UserNotFoundException(String message) {
         super(message);
     }
}
